server/repository/src/db_diesel/support_upload_row.rs

```diff
@@ -0,0 +1,112 @@
+use crate::{
+    ChangeLogInsertRow, ChangelogRepository, ChangelogTableName, RepositoryError, RowActionType,
+    StorageConnection, Upsert,
+};
+
+use chrono::NaiveDateTime;
+use diesel::prelude::*;
+use diesel_derive_enum::DbEnum;
+use serde::{Deserialize, Serialize};
+
+table! {
+    support_upload(id) {
+        id -> Text,
+        datetime -> Timestamp,
+        site_id -> Text,
```

Comment:
site_id is actually and integer I think, can you check other usages?

Reply:
Bigger questions actually, I think we'll either need to add a store_id to this table (and let them download the database for a store (which will include the entire site) or we'll need to make changes to sync logic to look at this site_id. I think changing to store_id is the simple solution for now...